metaserver/paymentrunner.go

```diff
@@ -0,0 +1,120 @@
+package metaserver
+
+import (
+	"fmt"
+	"skybin/core"
+	"time"
+)
+
+func (server *MetaServer) startPaymentRunner() {
+	// Frequency at which the runner should be triggered (should probably put this in a config file)
+	runnerFrequency := time.Minute * 1
+
+	// Ticker triggering the runner.
+	ticker := time.NewTicker(runnerFrequency)
+
+	go func() {
+		for range ticker.C {
+			server.logger.Println("Running payments...")
+			server.runPayments()
+		}
+	}()
+}
+
+func (server *MetaServer) runPayments() {
```

Comment:
Suggestions: 
1) Return an error from this method and do the logging in the parent. 
2) Don't make this a method no the server. Instead, take as the DB as an argument or place it in a struct ```type PaymentsRunner struct { DB metaserver.DB }```.  (maybe) test.
3) Use the same time.Time for the transaction Date and lastPaymentTime.